vector/environment.yml

```diff
@@ -10,5 +10,5 @@ dependencies:
 - rasterstats
 - requests
 - folium
-- seaborn
-- jupyter
\ No newline at end of file
+- matplotlib=2
```

Comment:
I fixed all the package upstream and this is not really needed, I only added it here to ensure mpl 2 will be installed.